docs/BreakingChanges/166 WCF transport security supports certificates stored using CNG.md

```diff
@@ -0,0 +1,52 @@
+## 166: WCF transport security supports certificates stored using CNG
+
+### Scope
+Minor
+
+### Version Introduced
+4.6.2
+
+### Source Analyzer Status
+Investigating
+
+### Change Description
+Starting with apps that target the .NET Framework 4.6.2, WCF transport security supports certificates stored using the Windows Cryptography Library (CNG). This support is limited to certificates with a public key that has an exponent no more than 32 bits in length. When an application targets the .NET Framework 4.6.2, this feature is on by default.
+
+In earlier versions of the .NET Framework, the attempt to use X509 certificates with a CSG key storage provider throws an exception. 
+- [X] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+Apps that target the .NET Framework 4.6.1 and earlier but are running on the .NET Framework 4.6.2 can enable support for CNG certificates by adding the following line to the `<runtime>` section of the app.config or web.config file:
+
+   ```xml
+   <runtime>
+      <AppContextSwitchOverrides value="Switch.System.ServiceModel.DisableCngCertificates=false" />
+   </runtime>
+   ```   
+
+This can also be done programmatically with the following code:
+
+   ```cs
+   private const string DisableCngCertificates = @"Switch.System.ServiceModel.DisableCngCertificate"; 
+   AppContext.SetSwitch(disableCngCertificates, false);
+   ```cs
```

Comment:
* Fix formatting by using \`\`\` to close a code block instead of ```cs
* May want to specify the language because when you view the formatted text, you can't tell which one is the C# code block and which is the VB block.